MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java

```diff
@@ -345,6 +351,24 @@
                 }
                 break;
             }
+            case CMD_BEGIN_EDUCATION: {
+                for (Person person : people) {
+                    EducationController.beginEducation(gui.getCampaign(), person, data[1], data[2], Integer.parseInt(data[3]), data[4], data[5]);
```

Comment:
In the previous step I converted an Integer into a String, so it could be parsed into `setActionCommand()`, this just reverts that step so should be a non-issue.